conf/cassandra.yaml

```diff
@@ -2368,3 +2368,111 @@ drop_compact_storage_enabled: false
 #   compatibility mode would no longer toggle behaviors as when it was running in the UPGRADING mode.
 #
 storage_compatibility_mode: NONE
+
+# Configuration for AutoRepair Scheduler.
+# auto_repair:
+#   # Enable/Disable the auto-repair scheduler.
+#   # If set to false, the scheduler thread will not be started.
+#   # If set to true, the repair scheduler thread will be created. The thread will
+#   # check for secondary configuration available for each repair type (full, incremental,
+#   # and preview_repaired), and based on that, it will schedule repairs.
+#   enabled: true
+#   repair_type_overrides:
+#     full:
+#       # Enable/Disable full auto-repair
+#       enabled: true
+#       # Minimum duration between repairing the same node again. This is useful for tiny clusters, such as
+#       # clusters with 5 nodes that finish repairs quickly. The default is 24 hours. This means that if the scheduler
+#       # completes one round on all nodes in less than 24 hours, it will not start a new repair round on a given node
+#       # until 24 hours have passed since the last repair.
+#       min_repair_interval: 24h
+#       token_range_splitter:
+#         parameters:
+#           # The target and maximum amount of bytes that should be included in a repair
+#           # assignment. This scopes the amount of work involved in a repair and includes
+#           # the data covering the range being repaired.
+#           bytes_per_assignment: 200GiB
+#           # The maximum number of bytes to cover in an individual schedule. This serves as
+#           # a mechanism to throttle the work done in each repair cycle. You may reduce this
+#           # value if the impact of repairs is causing too much load on the cluster or increase it
+#           # if writes outpace the amount of data being repaired. Alternatively, adjust the
+#           # min_repair_interval.
+#           max_bytes_per_schedule: 100000GiB
+#     incremental:
+#       enabled: true
+#       # Incremental repairs finish quickly, so they can be run more frequently. The default is 1 hour.
+#       min_repair_interval: 1h
+#       token_range_splitter:
+#         parameters:
+#           # Configured to attempt repairing 50GiB of data per repair.
+#           # This throttles the amount of incremental repair and anticompaction done per schedule
+#           # after incremental repairs are turned on.
+#           bytes_per_assignment: 50GiB
+#           # The maximum number of bytes to cover in an individual schedule to 100GiB.
+#           # Consider increasing max_bytes_per_schedule if
+#           # more data is written than this limit within the min_repair_interval.
+#           max_bytes_per_schedule: 100GiB
```

Comment:
Was internally debating whether we should also include `max_tables_per_assignment` and `partitions_per_assignment` just here under full repair so they are known, but then one thing I realized is that `partitions_per_assignment` is scaled by `repair_session_max_tree_depth`, which is actually deprecated!   Will think about that some more, but for now i think they are ok to omit here, as I don't think there's much value to tuning `max_tables_per_assignment`, maybe best to just leave it in the overview doc.

Reply:
Looks like these are already covered below in `global_settings` so you can ignore this, other than the part about `partitions_per_assignment` which I'll think some more about as we probably shouldn't base a property on another one thats already deprecated.